junit-mock/src/lib.rs

```diff
@@ -69,15 +70,30 @@ pub struct Options {
     pub test_rerun: TestRerunOptions,
 }
 
-#[derive(Debug, Parser)]
+impl Default for Options {
+    fn default() -> Self {
+        Options::try_parse_from([""]).unwrap()
+    }
+}
+
+#[test]
+fn options_can_be_defaulted_without_panicing() {
+    Options::default();
+}
```

Comment:
Removes the need for `unwrap`ing anytime we need to create options in tests or elsewhere